tools/podmanhtmltest.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env sh
+set -ex
+
+podman run --rm -ti \
+  -v "$(pwd):/test:Z" \
+  "${CHE_DOCS_IMAGE:-wjdp/htmltest}" \
```

Comment:
where is referenced CHE_DOCS_IMAGE ?

Reply:
It is not. It gives the ability to a user to change the image used.
But here it's a leftover of a copy-paste from another script.

Reply:
Fixed